config/e2e/rbac.yaml

```diff
@@ -161,6 +161,24 @@ rules:
     - get
     - list
     - watch
+    # for Elastic Agent in Fleet mode
+  - apiGroups:
+    - "coordination.k8s.io"
+    resources:
+    - leases
+    verbs:
+    - get
+    - create
+    - update
+    # for Elastic Agent Kubernetes integration
+  - apiGroups:
+    - "batch"
+    resources:
+    - jobs
```

Comment:
What's the batch job permissions for?

Reply:
This is for Kubernetes integration to fetch Job stats. Also present in https://raw.githubusercontent.com/elastic/beats/master/deploy/kubernetes/elastic-agent-managed-kubernetes.yaml.